actionview/test/template/url_helper_test.rb

```diff
@@ -404,6 +405,12 @@ def test_link_to_if_with_block
     assert_dom_equal %{<a href="/">Listing</a>}, link_to_if(true, "Listing", url_hash) { "Fallback" }
   end
 
+  def test_link_to_data_nil_equal
+    link_type1 = link_to('test', '#', 'class' => nil, 'data-tooltip' => nil)
+    link_type2 = link_to('test', '#', class: nil, data: {tooltip: nil})
```

Comment:
This test should be in the tag_helper_test file. Could you move it and use `content_tag` instead of `link_to`?